CDP4Composition/Views/CommonThingControl.xaml

```diff
@@ -24,7 +24,8 @@
     </UserControl.Resources>
 
     <dxb:ToolBarControl Height="30">
-        <dxb:BarSplitButtonItem Glyph="{dx:DXImage Image=Add_16x16.png}" 
+        <dxb:BarSplitButtonItem Command="{Binding CreateCommand}"
```

Comment:
What happens when there are multiple items in the below popup menu?

Reply:
Nothing. It will always call whatever the CreateCommand has been set to. The create command may also have been added to the popup menu.